_data/wmt_events.json

```diff
@@ -0,0 +1,212 @@
+       },
+       {
+         "title":"Title",
+         "description":"In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before the final copy is available."
+       }
+     ],
+     "languagePairs":[
+       "English - Czech",
+       "English - Russian",
+       "English - Armenian",
+       "English - Czech",
+       "English - Czech"
+     ],
+     "scientificPapers":[
+       {
+         "topics":[
+           "first topic",
+           "sec topic",
+           "third topic",
+           "4th topic"
+         ]
+       },
+       {
+         "researchPapers":[
+           "In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before the final copy is available.",
+           "In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before the final copy is available.",
+           "In publishing and graphic design, Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. Lorem ipsum may be used as a placeholder before the final copy is available."
+         ]
+       },
+       {
+         "systemPapers":[
+           "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+           "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+           "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+           "Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+         ]
+       }
+     ],
+     "paperSubmission":[
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+     ],
+     "posterFormat":[
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry."
+     ],
+     "multidaySchedule":[
+       [
+         {
+           "time":"10:30 - 11:00",
```

Comment:
I think we should keep it machine readable

So `startTime`, and then an optional `endTime`.

Or even just
```
[
    "10:30": "Some activity"
]
```
No end times.